phoenix-core/src/it/java/org/apache/phoenix/monitoring/PhoenixMetricsIT.java

```diff
@@ -232,6 +232,7 @@ private boolean verifyMetricsFromSinkOnce(Map<String, Long> expectedMetrics) {
                         long actualValue = metric.value().longValue();
                         if (expectedValue != actualValue) {
                             LOG.warn("Metric from Hadoop Sink: " + metric.name() + " didn't match expected.");
+                            LOG.warn("Expected: " + expectedValue + " Actual: " + actualValue);
```

Comment:
You can remove this log line or merge it into previous one, otherwise it will be difficult to comprehend when all these log lines are together.